tests/fr/droit_successions.catala_fr.result

```diff
@@ -135,7 +135,8 @@ champ d'application RéserveHéréditaire:
   définition patrimoine_assiette_réserve_héréditaire égal à
     patrimoine_total
     - (
-      somme argent de (
+      somme argent
```

Comment:
this is weird ? The previous version was better ?

Reply:
I guess it's fine, the new version is not that bad either.

Reply:
I'll try and add an exception for this particular case.

Reply:
I cannot manage to express this case.. I'll leave it as is for now